src/controller/userController.js

```diff
@@ -79,15 +69,14 @@ exports.login = async (req, res) => {
 
     if (passwordValidation) {
       // Generate JWT token
-      const token = jwt.sign(
-        { userId: userData._id, username: userData.userName },
-        process.env.SECRETKEY
-      );
+      const token = jwt.sign({ userId: userData._id }, process.env.SECRETKEY, {
+        expiresIn: "1d",
+      });
```

Comment:
create a separate file for JWT token operations like `sign` and `verify` file location --> `src/utils/jwt.js`

Reply:
resolved